conf/systemd/plackup_ledgersmb.service

```diff
@@ -0,0 +1,33 @@
+# Place this in /etc/systemd/system/plackup_ledgersmb.service; enable & start
+[Unit]
+Description=LedgerSMB Plack
+After=network.target
+
+[Service]
+Restart=always
+User=www-data
+WorkingDirectory=WORKING_DIR
+#
+ExecStart=/usr/bin/plackup                      \
+    --host ubuntu-vm                            \
+    --port 5000                                 \
+    --daemonize                                 \
+    --workers 5                                 \
+    --access-log /var/log/plackup_ledgersmb.log \
+    -I lib                                      \
+    -E development                              \
+    --server HTTP::Server::PSGI                 \
```

Comment:
Yes, fixed & pushed

Le 2016-04-22 14:55, Erik Huelsmann a écrit :
>
> In conf/systemd/plackup_ledgersmb.service
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60786665>:
>
> > +After=network.target
> > +
> > +[Service]
> > +Restart=always
> > +User=www-data
> > +WorkingDirectory=WORKING_DIR
> > +#
> > +ExecStart=/usr/bin/plackup                      \
> > +    --host ubuntu-vm                            \
> > +    --port 5000                                 \
> > +    --daemonize                                 \
> > +    --workers 5                                 \
> > +    --access-log /var/log/plackup_ledgersmb.log \
> > +    -I lib                                      \
> > +    -E development                              \
> > +    --server HTTP::Server::PSGI                 \
>
> This line and the next have development settings. I'm not too fond of
> that, because many people will simply copy things into place and not
> really look at what's been put inplace; can we put production config
> here by default?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60786665>
>